pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -662,29 +626,31 @@ def add_qiskit_data(
                     self.tkc.add_gate(
                         unitary_box,
                         list(reversed(qubits)),
-                        **condition_kwargs,
                     )
 
             elif optype == OpType.Barrier:
                 self.tkc.add_barrier(qubits)
 
             elif optype == OpType.CircBox:
                 circbox = _build_circbox(instr, circuit)
-                self.tkc.add_circbox(circbox, qubits + bits, **condition_kwargs)  # type: ignore
+                self.tkc.add_circbox(circbox, qubits + bits)  # type: ignore
```

Comment:
It is not really clear to me where this condition has gone? Can you give maybe some details?

Reply:
(Same for the two below)

Reply:
These `condition_kwargs` dictionaries were populated from the `Instruction.condition` property. In qiskit v2 this property is removed. Conditional gates in qiskit v2 are `IfElseOp` instances which have a `condition` property. These `IfElseOp` instructions have their own handling which was added in previous PRs.